gems/pending/Gemfile

```diff
@@ -18,7 +18,7 @@ gem "ezcrypto",                "=0.7",              :require => false
 gem "ffi",                     "~>1.9.3",           :require => false
 gem "ffi-vix_disk_lib",        "~>1.0.2",           :require => false  # used by lib/VixDiskLib
 gem "fog-openstack",           "~>0.1.7",           :require => false
-gem "hawkular-client",         "=2.3.0",            :require => false
+gem "hawkular-client",         "=2.4.0",            :require => false
```

Comment:
Of course it's better not to have the automation fail. 
But a preferred way would be to base this PR on the PR that updates the gem (and then rebase when that one is merged), or at least mark it with "depends on ..." like [this](https://github.com/ManageIQ/manageiq/pull/10287#issuecomment-238083841)